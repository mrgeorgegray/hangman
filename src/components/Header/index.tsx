const Header = () => (
  <header>
    <p>Hangman App</p>
    <hr />
  </header>
);

export default Header;
