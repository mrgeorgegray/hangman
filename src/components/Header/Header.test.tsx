import React from "react";
import { render } from "@testing-library/react";

import Header from ".";

describe("<Header />", () => {
  it("renders", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
