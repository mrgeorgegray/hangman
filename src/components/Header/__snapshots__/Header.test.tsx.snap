// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders 1`] = `
<div>
  <header>
    <p>
      Hangman App
    </p>
    <hr />
  </header>
</div>
`;
